manifests/a/AutoDesk/DWGTrueView/4.0.19.0/AutoDesk.DWGTrueView.yaml

```
@@ -0,0 +1,9 @@
+# Created using wingetcreate 1.0.4.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.1.0.schema.json
+
+PackageIdentifier: AutoDesk.DWGTrueView

```

```suggestion
PackageIdentifier: Autodesk.DWGTrueView
```
Publisher already exists as `Autodesk` - Note the capitalization